rpcs3/Emu/Cell/Modules/sceNp2.cpp

```diff
@@ -750,6 +750,22 @@ error_code sceNpMatching2GetRoomSlotInfoLocal(SceNpMatching2ContextId ctxId, con
 		return SCE_NP_MATCHING2_ERROR_NOT_INITIALIZED;
 	}
 
+	roomSlotInfo->roomId        = roomId;
+	roomSlotInfo->publicSlotNum    = 2; // TODO
+	roomSlotInfo->passwordSlotMask   = 0; // TODO
```

Comment:
This is missing some sort of check for `if (roomSlotInfo)`, either with a proper CELL return value or some exception for ease of debugging.

```suggestion
	roomSlotInfo->roomId             = roomId;
	roomSlotInfo->publicSlotNum      = 2; // TODO
	roomSlotInfo->passwordSlotMask   = 0; // TODO
```

Reply:
so such check exists in similar funcs and i dont think it's necessary

Reply:
The checks are missing because of lazyness as far as I can tell. Maybe just add a TODO instead